.github/workflows/build-linux-packages.yml

```diff
@@ -0,0 +1,94 @@
+name: Build Linux Packages
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*.*.*
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+  schedule:
+    - cron: '0 4 * * *'
```

Comment:
Is it necessary to do this on a schedule?

Reply:
Yes, there's a lot of dependencies and other moving parts and we need to know when it starts failing, which happens from time to time.